app/views/classify_concerns/new.html.erb

```diff
@@ -13,7 +13,11 @@
     <% (order + classify_concern.all_curation_concern_classes).uniq.each do |klass| %>
       <% if can? :create, klass %>
         <li class="work-type">
-          <h3 class="title"><%= klass.human_readable_type %></h3>
+          <% if klass == Etd %>
```

Comment:
@scherztc this view is just for Grad School staff (who know what 'ETD' means), plus the full 'Thesis and Dissertation' label is too long and messes up the wrapping.